news/559.bugfix

```diff
@@ -0,0 +1,3 @@
+Internationalize selectable columns in collection and tabular view.
+This fixes https://github.com/plone/Products.CMFPlone/issues/2597
+[vincentfretin]
```

Comment:
This is not going to be a `.bugfix` if the PR stays like this. Extension has to be reevaluated before merging.

Reply:
As the PR is now the extension should be `.breaking` because it is moving code from one place to another. I would really like to scale it down to `.feature` :)

Reply:
Ah moving the vocabulary code you mean in vocabulary.py inside the same package?
Do we consider this as breaking?

Reply:
I would say so... 
@jensens, semver jedi master, what do you think?

Anyway I see you added an import (which is fine). I will anyway probably revisit the code once this is merge to add a deprecation.

Reply:
deprecated import added